File: app/src/main/java/hu/bme/szn/randomjokes/interactor/jokes/event/DeleteJokeEvent.java
package hu.bme.szn.randomjokes.interactor.jokes.event;

public class DeleteJokeEvent {

    private Long id;
    private Throwable throwable;

    public DeleteJokeEvent() {
    }

    public DeleteJokeEvent(Long id, Throwable throwable) {
        this.id = id;
        this.throwable = throwable;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Throwable getThrowable() {
        return throwable;
    }

    public void setThrowable(Throwable throwable) {
        this.throwable = throwable;
    }
}

File: app/src/main/java/hu/bme/szn/randomjokes/interactor/jokes/event/DeleteJokeEvent.kt
package hu.bme.szn.randomjokes.interactor.jokes.event

import hu.bme.szn.randomjokes.model.Joke

data class DeleteJokeEvent(
        var code: Int = 2,
        var joke:Joke? = null,
        var throwable: Throwable? = null
)